test/jdk/tools/jpackage/share/JavaOptionsTest.java

```diff
@@ -36,7 +36,7 @@
  * @summary jpackage create image with --java-options test
  * @library /test/jdk/tools/jpackage/helpers
  * @build jdk.jpackage.test.*
```

Comment:
Helper classes will be build without `-Xlint:all -Werror`, right? If yes do we need to add `-Xlint:all -Werror` as well when building helper classes?

Reply:
> do we need to add -Xlint:all -Werror as well when building helper classes?

Good question. They are implicitly built by jtreg when requested through `@build jdk.jpackage.test.*` tag. I don't know how it constructs javac command line. @sormuras can you comment?